src/audio-worklet/migration/spn-recorder/index.njk

```diff
@@ -31,43 +85,35 @@ to_root_dir: ../../../
   download link and preview the recording. To restart the recording,
   please click the Refresh button.</p>
 
-<h3 class="text-2xl text-bold text-green-500" id="click-to-start">Click anywhere to start!</h3>
-
 <div class="demo-box">
-  <div class="Live">
-    <h2>Live</h2>
-    <div>
-      <button id="monitor">
-        Turn monitor <span>on</span>
-      </button>
-      <button id="viz-toggle">
-        <span>Pause</span> visualizer
-      </button>
-    </div>
-
-    <canvas id="live-canvas"></canvas>
-  </div>
-
   <div id="recording">
-    <h2>Recording</h2>
-
     <div>
       <p>Length: <span id="data-len">0</span>sec</p>
-      <button id="record">
+      <button disabled id="record">
         <span id="record-text">Start</span> Recording
       </button>
+      <button disabled id="stop">
+        Stop Recording
+      </button>
+      <a href="" id="download-link">
+        <button disabled id="download-button">Download File</button>
+      </a>
     </div>
 
-    <canvas id="recording-canvas"></canvas>
+    <div class="recording-display">
```

Comment:
Thanks for trying this idea out, but I think we need to implement the VU meter with `canvas`.

Reply:
Sure, I open another PR which is using the canvas to design the vu-meter